packages/dapp/src/hooks/userTracking/useUserTracking.ts

```diff
@@ -114,14 +109,17 @@ export function useUserTracking() {
         });
       }
       if (
-        !disableTrackingTool?.includes(EventTrackingTool.Raleon) &&
-        account.isActive &&
-        account.address
+        !disableTrackingTool?.includes(EventTrackingTool.Cookie3) &&
+        cookie3?.trackEvent({
+          category,
+          action,
+          name: label,
+          //value: todo add to trackingEventProps
```

Comment:
@Addminus Also, to keep track of USD a user has spent or received, we might consider using that "value" prop.
Currently, we don´t have it in place within our trackEvents, but we could possibly add it as an optional parameter to the ones, where we actually need them: e.g. Widget-Events (started / completed / failed / updated).

For now, it works without - but it could add value to the data for later analyzation.
Wdyt?

![image](https://github.com/lifinance/jumper.exchange/assets/43956540/0dabb5b4-6705-4d60-bb2d-f4b0b52b70cf)

Reply:
I have now also added an optional value-property to the trackEvent hook.